client/src/components/dashboard/DashBoard.js

```diff
@@ -1,50 +1,63 @@
-/* eslint require-jsdoc:0 */
+/* eslint require-jsdoc: "off"  */
+/* eslint class-methods-use-this: "off" */
 import React, { PropTypes } from 'react';
-import { connect } from 'react-redux';
 import toastr from 'toastr';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import 'froala-editor/js/froala_editor.pkgd.min';
 import 'froala-editor/css/froala_style.min.css';
 import 'froala-editor/css/froala_editor.pkgd.min.css';
 import 'font-awesome/css/font-awesome.css';
-import TextEditor from './textEditor';
-import Cards from './Cards';
+import TextEditorComponent from './textEditor';
+import CardsComponent from './Cards';
 import * as docActions from '../../actions/docActions';
 import * as userActions from '../../actions/userActions';
 
-class DashBoard extends React.Component {
-  constructor(props) {
-    super(props);
+/**
+ * Class to create dashboard component
+ */
+export class DashBoard extends React.Component {
+  componentDidMount() {
+    // Ensure user is authenticated before loading uer details
+    if (this.props.isAuthenticated) {
```

Comment:
**Nitpick**
Do same as above.
If you can, you should create a `requireAuth` Higher Order Component that will validate that a user is  authenticated before accessing protect routes.